lib/SILOptimizer/FunctionSignatureTransforms/ExistentialSpecializer.cpp

```diff
@@ -257,6 +258,12 @@ bool ExistentialSpecializer::canSpecializeCalleeFunction(ApplySite &Apply) {
   if (Callee->getInlineStrategy() == Inline_t::AlwaysInline)
     return false;
 
+  /// Ignore externally linked functions with public_external or higher
+  /// linkage.
+  if (isAvailableExternally(Callee->getLinkage())) {
```

Comment:
This fixes the swiftpm bug related to external linkage functions causing name mangling issues.